blog/guide/README.md

```diff
@@ -10,17 +11,17 @@
 
 
 ## مفاهیم و تعاریف
-قبل از اینکه بیشتر وارد جزئیات فندق و استفاده از فندق بشیم لازم هست تا چند مفهوم مرتبط با فندق رو با هم مرور کنیم:
+قبل از اینکه بیشتر وارد جزئیات فندق و روش استفاده از اون بشیم لازم هستش تا چند مفهوم مرتبط با فندق رو با هم مرور کنیم:
 * **کلاینت فندق**   
-کلاینت فندق یک نرم‌افزار تحت کامندلاین است که به کاربران کمک می‌کنه از فندق استفاده کنند.
-* **Image**   
-در فندق به هر سرویس یک Image می‌گوییم و شما میتونید هر Image رو معادل یک Image در داکر تصور کنید، به ازای هر Image یک یا چند سرویس در فندق اجرا می‌شوند.
+کلاینت فندق یک نرم‌افزار تحت خط فرمان  (Command Line) است که به کاربران کمک می‌کنه از فندق استفاده کنند.
+* **Image**   در فندق به هر سرویس یک Image می‌گوییم و شما میتونید هر Image رو معادل یک Image در داکر تصور کنید.
```

Comment:
the definition is not correct. 
Each image is a program that can be executed on fandogh as a service.